src/base/core/int.c

```diff
@@ -864,6 +864,7 @@ static int int15(void)
     case 0x81:			/* default BIOS hook: device close */
     case 0x82:			/* default BIOS hook: program termination */
 	HI(ax) = 0;
+	/* Fall through */
```

Comment:
I think break is actually missing here.
Setting AH to 0 and CARRY at the
same time, probably makes no sense.

Reply:
Agreed, see https://fd.lod.bz/rbil/interrup/bios/1582.html

Reply:
Thanks, I'll fix that up!